scripts/close_inactive.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require 'octokit'
+
+REPOS = ["theforeman/foreman"]
+KEEP = 6 # months
```

Comment:
> These configuration items should be in a config file like the other code.

Is there any reason to do that (e.g. existing Puppet module that
configures this)? Otherwise I doubt this is worth any effort, we are not
going to change this often.

-- 
Later,
 Lukas #lzap Zapletal